lib/editor/components/HourMinuteInput.js

```diff
@@ -81,7 +82,13 @@ export default class HourMinuteInput extends Component<Props, State> {
           onChange={this._onChange}
           onFocus={this._onFocus}
           placeholder={'hh'}
-          style={{ width: '5.5ch', borderRight: 'none', ...style }}
+          style={{
+            width: '5.5ch',
+            borderRight: 'none',
```

Comment:
Sort CSS

Reply:
Sorted in https://github.com/ibi-group/datatools-ui/pull/892/commits/69ed91b487e9585118c726639dd711c98d0c2df1